### ISSUE TITLE
Unable to refresh history data using refreshHistory method in React component

### ISSUE THREAD

**Issue by samsam002** [ONLYOFFICE/document-editor-react#42]
I am using the OnlyOffice component in a React project and encountering an issue when trying to refresh the history data using the refreshHistory method. The method is being called within the onRequestHistory callback, but it throws a TypeError indicating that refreshHistory is not a function.
 Here's the relevant code snippet:

```
onRequestHistory: async (event) => {
  console.log('Requesting history data...', event);
  const data = await get(`/contracts/${itemId}/history`);
  event?.target?.docEditor.refreshHistory({
    currentVersion: data.currentVersion,
    history: data.history,
  });
},

```
The error message received is:

`TypeError: editor.refreshHistory is not a function`

I have tried accessing the refreshHistory method through event.target.docEditor, but it doesn't seem to be available.

Additional context:

I am using the OnlyOffice component in a React project.
The OnlyOffice component is initialized with the necessary props and configurations.
The onRequestHistory callback is triggered when the history data needs to be refreshed.
I have followed the documentation and examples provided for integrating OnlyOffice in a React project.
I would appreciate any guidance or suggestions on how to properly refresh the history data using the OnlyOffice component in a React environment. If there are any changes in the API or recommended approaches, please let me know.

in other words : 
From where i can get the docEditor instance in the document-editor-react implementation.

**Comment by aleksandrfedorov97** [comment]
Hi [samsam002](https://github.com/samsam002), you can get the docEditor instance like this:
`var docEditor = window.DocEditor.instances[id]`
where `id` is the [сomponent unique identifier](https://github.com/ONLYOFFICE/document-editor-react?tab=readme-ov-file#api).

**Comment by trofim24** [comment]
A complete version history setup:

```jsx
import { DocumentEditor } from "@onlyoffice/document-editor-react";

const EDITOR_ID = "docxEditor";

export default function Editor({ config, itemId }) {
  const onRequestHistory = async () => {
    const data = await get(`/contracts/${itemId}/history`);

    window.DocEditor.instances[EDITOR_ID].refreshHistory({
      currentVersion: data.currentVersion,
      history: data.history,
    });
  };

  // Fired when the user picks a version in the history panel.
  const onRequestHistoryData = async (event) => {
    const version = event.data; // the version number
    const data = await get(`/contracts/${itemId}/history/${version}`);

    window.DocEditor.instances[EDITOR_ID].setHistoryData(data);
  };

  const onRequestHistoryClose = () => {
    document.location.reload();
  };

  return (
    <DocumentEditor
      id={EDITOR_ID}
      documentServerUrl={DOCUMENT_SERVER_URL}
      config={config}
      events_onRequestHistory={onRequestHistory}
      events_onRequestHistoryData={onRequestHistoryData}
      events_onRequestHistoryClose={onRequestHistoryClose}
    />
  );
}
```

Two things to watch out for:

- The **Version History** button only appears if the `onRequestHistory` handler was present when the editor was initialized (internally: `canUseHistory = !!events.onRequestHistory`). Same for `onRequestHistoryClose` and `onRequestRestore`. The config is read once at startup, so pass these props from the first render - adding them later has no effect until the editor is recreated.
- The payloads for [`refreshHistory`](https://api.onlyoffice.com/docs/docs-api/usage-api/methods/#refreshhistory) and [`setHistoryData`](https://api.onlyoffice.com/docs/docs-api/usage-api/methods/#sethistorydata) are described in the Methods reference, and the whole flow is walked through in [Document history](https://api.onlyoffice.com/docs/docs-api/get-started/how-it-works/document-history/). If the history object contains local URLs, remember the token.
